ArduCopter/mode_loiter_poi.cpp

```diff
@@ -0,0 +1,217 @@
+#include "Copter.h"
+
+#if MODE_LOITER_ENABLED == ENABLED
+
+/*
+ * Init and run calls for loiter flight mode
+ */
+
+// loiter_init - initialise loiter controller
```

Comment:
Match the comment to the entity.

Reply:
Modified at [6b260ba3](https://github.com/ArduPilot/ardupilot/pull/18063/commits/6b260ba3df71216404ef88db8f60c656f9a4f597) Do I have to rebase to merge all commits?